packages/devtools_app/lib/src/screens/deep_link_validation/project_root_selection/select_project_view.dart

```diff
@@ -141,6 +134,15 @@ class _SelectProjectViewState extends State<SelectProjectView>
         gac.AnalyzeFlutterProject.flutterProjectSelected.name,
       );
     }
+
+    if (androidVariants.isEmpty && iosBuildOptions.configurations.isEmpty) {
+      ga.select(
+        gac.deeplink,
+        gac.AnalyzeFlutterProject.flutterInvalidAndroidProjectSelected.name,
+      );
+      await showNonFlutterProjectDialog();
+      return ;
```

Comment:
remove semi colon

Reply:
I guess you mean to remove the space before the semicolon. updated.